python/Ganga/test/GPI/FileTests/TestSharedFileClient.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import absolute_import
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+from Ganga.testlib.file_utils import generate_unique_temp_file
+
+import datetime
+import time
+import os
+import shutil
+
+class TestSharedClient(GangaUnitTest):
```

Comment:
These tests seem to be copy-pasted from `MassStorageFile`. It's not surprising that they are so similar but in that case they should use a single, parametrised test.